huaweicloud/services/acceptance/fgs/resource_huaweicloud_fgs_function_test.go

```diff
@@ -42,7 +42,7 @@ func TestAccFgsV2Function_basic(t *testing.T) {
 				Config: testAccFgsV2Function_basic(randName),
 				Check: resource.ComposeTestCheckFunc(
 					rc.CheckResourceExists(),
-					resource.TestCheckResourceAttr(resourceName, "functiongraph_version", "v1"),
+					resource.TestCheckResourceAttr(resourceName, "functiongraph_version", "v2"),
```

Comment:
the default value is `v1` in `cn-north-4`, please revert it and append a comment right of this check step.